addon/templates/components/polaris-action-list/item.hbs

```diff
@@ -13,7 +13,7 @@
         role="presentation"
         class="Polaris-ActionList__Image"
         style={{imageBackgroundStyle}}
-      ></div>
+      />
```

Comment:
isn't div not supposed to be a self closing tag?

Reply:
Yeah I think it shouldn't be self-closing 🤷‍♂️ Does the linter complain if this is left as it was before @vladucu?

Reply:
lets disable the check here if it complains about it instead of making this change 🙂